bookstore/handlers.py

```diff
@@ -20,24 +20,35 @@ class BookstoreVersionHandler(APIHandler):
     """Handler responsible for Bookstore version information
 
     Used to lay foundations for the bookstore package. Though, frontends can use this endpoint for feature detection.
+
+    Methods
+    -------
+    get(self)
+        Provides version info and feature availability based on serverside settings.
+    build_response_dict(self)
+        Helper to populate response.
     """
 
     @web.authenticated
     def get(self):
-        self.finish(
-            json.dumps(
-                {
-                    "bookstore": True,
-                    "version": self.settings['bookstore']["version"],
-                    "validation": self.settings['bookstore']["validation"],
-                }
-            )
-        )
+        """GET /api/bookstore/
+
+        Returns version info and validation info for various bookstore features.
+        """
+        self.finish(json.dumps(self.build_response_dict()))
+
+    def build_response_dict(self):
+        """Helper for building the version handler's response before serialization."""
+        return {
+            "bookstore": True,  # TODO: Should we remove this; isn't it equivalent to the endpoint responding?
```

Comment:
I don't know what this field even represents. What is it used for?

Reply:
I think it was originally a placeholder as a response indicating that indeed the endpoint was from bookstore. 

I want to tackle that in a different PR though. Check out & comment on #132 please!

Reply:
I made this change in bfeecf1